### PR TITLE
Fixed Spanish Traduction

### PR DESCRIPTION
**Pull request by Block747**
Fixed the issues of the other file

### CODE REVIEW

File: locales/ES_es.json [hunk 1]
```diff
@@ -153,13 +153,13 @@
 			},
 			{
 				"name": "Stary",
-				"caption": "3DS dev and NEX dissector",
+				"caption": "Desarrollador de 3DS y disector NEX",
```
Comment (Block747) [inline locales/ES_es.json:156]: Now has got capital letters

File: locales/ES_es.json [hunk 2]
```diff
@@ -111,7 +111,7 @@
 			},
 			{
 				"name": "quarky",
-				"caption": "BOSS research and patch development",
+				"caption": "Investigador de BOSS y desarrollador de parches",
```
Comment (Block747) [inline locales/ES_es.json:114]: I didn't had context about BOSS so now i changed it

**Comment by ashmonty** [comment]
Thanks for the contribution, but as I said in the other pull request, there's already a localization in the dev branch... [ES_es.json](https://github.com/PretendoNetwork/website/blob/dev/locales/ES_es.json)

**Comment by ashmonty** [comment]
For that reason, I'm closing this, as it is redundant

**Pull request closed without merging**